### PR TITLE
Edit explanation for guidance hint & mandatory response

### PR DESCRIPTION
**Pull request by exprin**

### CODE REVIEW

**Comment by exprin** [comment]
Hi @joshuaberetta, I have made some minor changes to this article especially on the explanation for guidance hint and mandatory response. I just thought it's something you should be able to quickly review and commit.

File: source/question_options.md [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
 
 The Data Column Name (ID) is important because it is used in the column headers of tables and spreadsheets after your data has been collected. If you want your spreadsheet to follow a specific naming convention, you should specify the name for each of your questions before deploying the form as a data collection project.
 
 ## Guidance Hint (optional)
 
-**Guidance Hint** are help texts that will be displayed underneath your questions on the form.
+**Guidance Hints** are extra instructions that you can add to your questions as notes. These notes are not normally shown on the form by default.
 
-Hints are optional. The hint text is often used to provide additional instructions to your interviewer staff. For example, your question label might be `How old are you?` and your hint might be `If respondent doesn't know, enter 999`.
+Guidance hints can be used as internal notes when collaborating with others in developing the form. You can also show them on printouts or during training for enumerators.
```
Comment (joshuaberetta) [inline source/question_options.md:19]: It seems we are confusing the difference between the regular [`hint`](https://xlsform.org/en/#regular-hints) and [`guidance_hint`](https://xlsform.org/en/#guidance-hints) here now. This is the difference in how the user or enumerator will see them:
hint

I would suggest being explicit about which type of hint you are referring to and maybe include an XLSForm survey example such as this to exemplify the difference:

**survey**

| type | name | label              | hint           | guidance_hint           |
| ---  | ---  | ---                | ---            | ---                     |
| text | name | What is your name? | This is a hint | This is a guidance hint |

File: source/question_options.md [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
 
 ## Mandatory Response
 
-If a question is **mandatory**, the interviewer needs to provide an answer in order to finalize the form.
+This settings allows you to specify whether the question must be responded to at all times or not.
```
Comment (joshuaberetta) [inline source/question_options.md:23]: Maybe just note that this is equivalent to the `required` field in XLSForm and link to the docs: https://xlsform.org/en/#required

File: source/question_options.md [hunk 3]
```diff
@@ -1,37 +1,43 @@
...
+
+Three options are available:
+
+1. Yes - The question must be answered at all times. If the user does not provide a response, he/she will not be able to move to the next question or save the form.
```
Comment (joshuaberetta) [inline source/question_options.md:27]: Use "they" rather than "he/she"

File: source/question_options.md [hunk 4]
```diff
@@ -1,37 +1,43 @@
...
+2. No - The question is not mandatory and hence can be manually skipped
+3. Custom logic - You can define logic using XLSForm code which will define when the question will be mandatory. For example, if you set the following custom logic `age > 18`, the question will be mandatory when a preceding question with the data column name `age` is greater than 18
```
Comment (joshuaberetta) [inline source/question_options.md:29]: This would need to be `${age} > 18`

**Review by joshuaberetta — CHANGES_REQUESTED** [review]

**Comment by exprin** [comment]
Thanks so much for the comments @joshuaberetta. It seems I jumped the gun on several things, I really appreciate the direction. I will quickly work on the changes and resubmit

**Comment by exprin** [comment]
Hi @joshuaberetta, I have added a new commit to this article addressing the issues you mentioned and expanding it further. Please check.

**Review by joshuaberetta — APPROVED** [review]

**Pull request merged**